Games/types/Mafia/effects/Insanity.js

```diff
@@ -6,6 +6,14 @@ const { rlyehianify } = require("../../../../lib/TranslatorRlyehian");
 module.exports = class Insanity extends Effect {
   constructor() {
     super("Insanity");
+    this.listeners = {
+      death: function (player) {
```

Comment:
do you want all effects to be removed on death, or just insanity in particular?

rend didn't like removing all effects on death